plugins/org.jboss.tools.openshift.common.ui/src/org/jboss/tools/openshift/internal/common/ui/utils/GTK3Utils.java

```diff
@@ -55,5 +56,23 @@ private static boolean isGTK3Env() {
 		return !"0".equals(gtk3); //$NON-NLS-1$
 	}
 
+	/**
+	 * In GTK3 table viewer in some cases (e.g. if it is invisible,
+	 * on the next wizard page, and has vertical scroll) is not updated
+	 * after its content is changed even with refresh(true).
+	 * This method makes it to update in a radical way. It will 
+	 * take effect if invoked after tree of widgets including 
+	 * the table gets visible (e.g. on wizard page activated).
+	 * @param viewer
+	 */
```

Comment:
Added this to the javadoc (though github did not marked previous comment as obsolete).
@fbricon , so this issue is still actual for Mars?